src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -64,7 +65,8 @@ const Stream = React.createClass({
       multiSelected: false,
       anySelected: false,
       statsPeriod: this.props.defaultStatsPeriod,
-      realtimeActive: Cookies.get('realtimeActive') === 'true',
+      realtimeActive: Cookies.get('realtimeActive') === 'true' ||
+        (project && !project.firstEvent),
```

Comment:
hmm i'm wondering if we should check to see if there's a `false` cookie here too. like, how weird would it be for people if they pause it, click on the sample (or otherwise navigate away) and then come back and it's been un-paused? i'll defer to @ehfeng since he requested it.